### DISCUSSION TITLE
Source in Metrics should get a reset

### DISCUSSION THREAD

**Discussion by flowerhorse** [brunoherbelin/vimix#75] [Ideas]
Metrics with Source is very practical via the direct setting option. 
Analogous to Source Color Correction, the words should also have a reset function. 
Angle should, like the other numbers, also be editable to two decimal places via the keyboard for very fine adjusting. I had this case yesterday..

Scale: Scaling in free proportion is better done in another way than via the scale input.
Instead, here would be a good place for scaling to scale with a value, I think.

Are the values in Source Color Correction also editable?

![Screenshot_20230407_055714](https://user-images.githubusercontent.com/125514250/230539413-200c93d7-14b5-46d7-a674-8e9e60ed5946.png)

**Comment by brunoherbelin** [comment]
Aha ! yes I forgot to update this window !! Thanks for pointing this out
And yes, the precision was not good and the interface not that useful.
I also added a mode to link scale X and Y (keep aspect ratio)

Beta is now updated to take this into account. thanks!

**Comment by flowerhorse** [comment]
Great. I'll update right now and try it out tonight..
Will ProRes support remain in the beta flatpak?
Otherwise I'd rather do it when I have the video ready for the exhibition..

**Reply by flowerhorse** [reply]
The small Source menu has also gained in importance. By the fact that it makes number input possible in all ranges it extends the possibilities of the F2, F4, F5 menu. likewise resets of individual factors are now possible there, which was not before. and over the alpha reset can be switched even with selection immediately videos. 

Perhaps a switch would be missing here 'Reset Position' with value alpha zero, so that one can switch off here also a medium just as fast? 

When moving to 'free position' I had difficulties to move the window. It takes me several attempts, before the mouse has found the switch sensitive position, where m,an can move the window.

The moving of the window is important because you can adjust with the mouse over the window. To the left side it goes quite well but at the right side is with me screen end, there go only some pixels.
If one enters there over the mouse one must shift inevitably.
Perhaps it would be better from therefore if the window would generally stand on 'free positon'.

**Comment by brunoherbelin** [comment]
I was also not satisfied with this source editing and its position. I created a new toolbox separate from Metrics.

There is now a 'Source editor' toolbox, which can be configured to be at the bottom or top of the window, or freely moved, and auto-hidden when no source is selected. It is enabled from the submenu in the list of windows ;

![Screenshot from 2023-04-16 14-56-01](https://user-images.githubusercontent.com/60393189/232313073-ef22e2cc-b3c2-4821-822b-d477eb95a890.png)
